### PR TITLE
modified navbar, searchbar, panic button 

### PR DESCRIPTION
**Pull request by Julie-Salazar**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**
modified navbar, searchbar, panic button according to new specs for homepage (should work on all other pages as well)
see provided reference site from starick meeting 

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**
for compositional style: https://luma.org.au/services/family-and-domestic-violence/

future changes will follow the specs >>
Stuff they especially liked:
minimalist design
Their organisation of information
Their placement of the exit button and their footer.
The downloadable referral form and brochure included
How they did the headers (images appearing on hover)

for more info refer to pinned messages in cfc #starick channel

### CODE REVIEW

File: src/components/Home/core-service-item.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import Image, { StaticImageData } from "next/image";
+import React from "react";
```
Comment (cjnbennett) [inline src/components/Home/core-service-item.tsx:1]: Next.js automatically imports React for you, so you don't need to include this line

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,14 +1,14 @@
 import CoreServiceItem from "@/components/Home/core-service-item";
 import Subtitle from "@/components/Home/subtitle";
-
-import CounsellingServices from "~/images/home/counselling-services.png";
-import SafeAtHomeProgram from "~/images/home/safe-at-home-program.png";
-import SupportInPoliceStations from "~/images/home/support-in-police-stations.png";
-import TransitionalHousing from "~/images/home/transitional-housing.png";
-import TwoRefuges from "~/images/home/two-refuges.png";
+import React from "react";
```
Comment (cjnbennett) [inline src/pages/index.tsx:3]: Same as above

File: tsconfig.json [hunk 3]
```diff
@@ -19,6 +19,12 @@
       "~/*": ["./public/*"]
     }
   },
-  "include": ["next-env.d.ts", "./src/index.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "./src/index.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "**/*.svg.d.ts"
```
Comment (cjnbennett) [inline tsconfig.json]: I don't think this line is necessary because there are no `.svg.d.ts` files

**Review by cjnbennett — APPROVED** [review]
Looks like there's a merge conflict that you'll have to resolve, but otherwise good to merge
(Some minor pointers below)

**Review by Julie-Salazar — COMMENTED** [review]
deleted unecessary lines

**Review by seraph1201 — APPROVED** [review]
lgtm

**Review by seraph1201 — COMMENTED** [review]
lgtm

**Pull request merged**